mails/themes/modern/components/footer.html.twig

```diff
@@ -39,7 +39,7 @@
                             </tr>
                             <tr>
                               <td align="center" style="font-size:0px;padding:10px 25px;padding-top:0;word-break:break-word;">
-                                <div style="font-family:Open sans, arial, sans-serif;font-size:12px;line-height:25px;text-align:center;color:#656565;"> Powered by <a href="http://www.prestashop.com/" style="color:#656565;font-weight:400;">PrestaShop</a>
+                                <div style="font-family:Open sans, arial, sans-serif;font-size:12px;line-height:25px;text-align:center;color:#656565;"> Powered by <a href="https://www.prestashop.com/?utm_source=marchandprestashop&amp;utm_medium=e-mail=utm_campaign=footer_1-7" style="color:#656565;font-weight:400;">PrestaShop</a>
```

Comment:
Same here.

Reply:
This was is not translated (in modern theme, the shop link and the prestahop have been separated), so I'm not sure it is necessary